### DISCUSSION TITLE
How to connect to remove PC in another country?

### DISCUSSION THREAD

**Discussion by RomanAlxndrvch** [rustdesk/rustdesk#7937] [Q&A]
![image](https://github.com/rustdesk/rustdesk/assets/89692499/872c696d-0955-43c5-a06d-9a9aeec8014e)

Today I tried to connect to my PC in another country, but I didn't have access to do that. Do I have to play with some settings on my remote PC to keep using it as it was?

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1787689958276472982
